src/standalone/Vegetation/PlantHydraulics.jl

```diff
@@ -148,32 +149,50 @@ struct PlantHydraulicsParameters{
     conductivity_model::CP
     "Water retention model and parameters"
     retention_model::RP
-    "Root distribution function P(z)"
-    root_distribution::F
+    "Rooting depth parameter (m) - a characteristic depth below which 1/e of the root mass lies"
+    rooting_depth::RD
 end
 
 function PlantHydraulicsParameters(;
     ai_parameterization::PrescribedSiteAreaIndex{FT},
     ν::FT,
     S_s::FT,
-    root_distribution::Function,
+    rooting_depth::Union{Nothing, FT, ClimaCore.Fields.Field} = nothing,
+    root_distribution::Union{Nothing, Function} = nothing,
     conductivity_model,
     retention_model,
 ) where {FT}
-    return PlantHydraulicsParameters{
-        FT,
-        typeof(ai_parameterization),
-        typeof(conductivity_model),
-        typeof(retention_model),
-        typeof(root_distribution),
-    }(
-        ai_parameterization,
-        ν,
-        S_s,
-        conductivity_model,
-        retention_model,
-        root_distribution,
-    )
+    if root_distribution isa Nothing
+        return PlantHydraulicsParameters{
+            FT,
+            typeof(ai_parameterization),
+            typeof(conductivity_model),
+            typeof(retention_model),
+            typeof(rooting_depth),
+        }(
+            ai_parameterization,
+            ν,
+            S_s,
+            conductivity_model,
+            retention_model,
+            rooting_depth,
+        )
+    else
+        return PlantHydraulicsParameters{
+            FT,
+            typeof(ai_parameterization),
+            typeof(conductivity_model),
+            typeof(retention_model),
+            FT,
+        }(
+            ai_parameterization,
+            ν,
```

Comment:
```suggestion
        rooting_depth = if root_distribution isa Nothing
            rooting_depth
        else
            FT(1.0 / root_distribution(0.0))
        end
        return PlantHydraulicsParameters{
            FT,
            typeof(ai_parameterization),
            typeof(conductivity_model),
            typeof(retention_model),
            typeof(rooting_depth),
        }(
            ai_parameterization,
            ν,
            S_s,
            conductivity_model,
            retention_model,
            rooting_depth,
        )
```
Couldn't we write this like this instead?